collector/os_release.go

```diff
@@ -169,6 +178,12 @@ func (c *osReleaseCollector) UpdateStruct(path string) error {
 	} else {
 		c.version = 0
 	}
+
+	c.supportEnd, err = time.Parse("2006-01-02", c.os.SupportEnd)
+	if err != nil {
+		return err
```

Comment:
`SUPPORT_END` is an optional field, I think this will fail if there is none. I think we need to wrap this in an `if c.os.SupportEnd != ""`.